DFC.ServiceTaxonomy.GraphSync/Permissions.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System.Collections.Generic;
+using System.Threading.Tasks;
+using OrchardCore.Security.Permissions;
+
+namespace DFC.ServiceTaxonomy.GraphSync
+{
+    public class Permissions : IPermissionProvider
+    {
+        public static readonly Permission AdministerGraphs = new Permission("AdministerGraphs", "Administer graphs");
```

Comment:
Could do with extracting the hard-coded values in permissions into a central class so we can use and reference them elsewhere to avoid typos and promote reuse

Reply:
If you look at an [example of checking the permission](https://github.com/SkillsFundingAgency/dfc-servicetaxonomy-editor/pull/148/files#diff-81a28a57b72e94971dd37ea003e4e3e6), notice that the class itself is used, rather than the strings.

The strings are only for display in the Security admin section.